vaadin-grid-flow-parent/vaadin-grid-flow/src/test/java/com/vaadin/flow/component/grid/GridColumnGroupTest.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2000-2017 Vaadin Ltd.
```

Comment:
nit
```suggestion
 * Copyright 2000-2021 Vaadin Ltd.
```

Reply:
Thanks! Fixed.

I don't need to update copyright in other files because it concerns only new files, right?

Reply:
Only new files for now